src/MapView/MapView.web.tsx

```diff
@@ -7,7 +7,7 @@ import 'mapbox-gl/dist/mapbox-gl.css';
 import Direction from './Direction';
 import {DEFAULT_INITIAL_STATE} from './CONST';
 
-const MapView = forwardRef<MapViewHandle, MapViewProps>(function MapView(
+const MapView = forwardRef<MapViewHandle, Omit<MapViewProps, "isFocused">>(function MapView(
```

Comment:
This might be irrelevant depending on what we're going to do with my suggestion [here](https://github.com/Expensify/react-native-x-maps/pull/32/files#r1306696791), but here is a general tip:

All component specific implementations should have exactly the same interface. Instead of requiring `isFocused` for mobile and when omitting the prop on web, we can make `isFocused` an optional prop.

On web, we don't use it. On mobile, we use it. (but I also know `isFocused` is indeed needed on mobile for `MapView`to correctly work. And that's why I suggested [this](https://github.com/Expensify/react-native-x-maps/pull/32/files#r1306696791))